source/js/plugins/pixel-wrapper.js

```diff
@@ -76,25 +76,26 @@ export class PixelWrapper
 
     /**
      *  Creates the number of required layers based on the number of input ports in the Rodan job.
-     *  The variable numberInputLayers is defined in the outermost index.html 
+     *  The variable numberInputLayers is actually defined in the outermost index.html 
      */
     createLayers ()
     {
         // Set default tool to rectangle (for select region layer)
         this.pixelInstance.tools.currentTool = "rectangle";
 
         // Only create default layers once
-        if (this.layers.length !== 1) {
+        if (this.layers.length !== 1) 
             return;
-        }
 
-        let numLayers = numberInputLayers;
+        let numLayers = numberInputLayers; // jshint ignore:line
```

Comment:
What is the issue here?

Reply:
`numberInputLayers` is defined in the index.html, so the variable doesn't exist and throws the JSHint error.